content/get-started/climate-data-and-other-data-on-caladapt.md

```diff
@@ -22,12 +22,12 @@ The main datasets hosted on Cal-Adapt are listed in the table below. For more in
 | Derived Products                                  | Datasets created from LOCA, Historical gridded data, VIC runs and Wildfire simulations for Cal-Adapt tools.                                                                                                                 | <a href="https://www.energy.ca.gov/sites/default/files/2019-11/Projections_CCCA4-CEC-2018-015_ADA.pdf" target="_blank">Geospatial Innovation Facility</a>           | Maximum temperature, Minimum temperature, Precipitation, Evapotranspiration, Baseflow, Runoff, Soil moisture, Area burned probability                                                                                                                                                                                                                             | Observed historical, Modeled historical, RCP 4.5, RCP 8.5                     | 1950-2013, 2006-2100 |
-| CalFloD-3D (50 mt. coastal flooding surfaces)     | Inundation layers representing coastal flooding scenarios caused by sea level rise, storm surge and inland flooding due to rainfall under various climate change scenarios.                                                 | <a href="https://www.energy.ca.gov/sites/default/files/2019-11/Energy_CCCA4-CEC-2018-012_ADA.pdf" target="_blank">Radke et al. (2018)</a>                           | Inundation depth                                                                                                                                                                                                                                                                                                                                                  | RCP 4.5, RCP 8.5                                                              | 2000-2100            |
+| CalFloD3D-TFS (50 mt. coastal flooding surfaces)     | Inundation layers representing coastal flooding scenarios caused by sea level rise, storm surge and inland flooding due to rainfall under various climate change scenarios.                                                 | <a href="https://www.energy.ca.gov/sites/default/files/2019-11/Energy_CCCA4-CEC-2018-012_ADA.pdf" target="_blank">Radke et al. (2018)</a>                           | Inundation depth                                                                                                                                                                                                                                                                                                                                                  | RCP 4.5, RCP 8.5                                                              | 2000-2100            |
```

Comment:
I think it would help if we had CalFloD3D-TFS follow CalFloD-3D, it seems odd to have them spaced apart in the table.

Reply:
Yes, I am planning on rearranging this table when I update it in a separate PR.